tests/test_files/calculator/calculator.py

```diff
@@ -1,5 +1,24 @@
-# TODO: Implement a calculator program, that can add, subtract, multiply and divide two numbers.
+# Implement a calculator program, that can add, subtract, multiply and divide two numbers.
 #  - Numbers and operators given from the user as the command line arguments.
 #  - Consider using argpasre library.
 #  - Consider using eval builtin function.
 #  - Print or log the result.
+import argparse
+
+def check_number(value):
+    try:
+        float(value)
+        return value
+    except ValueError:
+        raise argparse.ArgumentTypeError("Argument is not a number")
+
+parser = argparse.ArgumentParser()
```

Comment:
Please try to outsource the parser into a separate def, that returns parser.parse_args()